.github/workflows/bundle-size.yml

```diff
@@ -0,0 +1,23 @@
+name: Compressed Size
+
+on:
+    pull_request:
+    workflow_dispatch:
+
+jobs:
+    build:
+        name: Bundle size check
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: Use desired version of NodeJS
+              uses: actions/setup-node@v2
+              with:
+                  node-version-file: '.nvmrc'
+                  cache: npm
+            - uses: preactjs/compressed-size-action@v2
```

Comment:
@fashuvo Based on our discussion regarding the security concern, let's lock `preactjs/compressed-size-action` to the latest commit SHA. So it would be `preactjs/compressed-size-action@b03ec15e1a93cb5c062d66a1c263f9ddbd2f81ef`.

This prevents the action from including newer changes during our run. And if we want to update a new version, we can do an audit of the changes made after this commit and update the commit SHA, which is much safer.

Reply:
I have updated the check to use the latest commit SHA.